sysutils/shmux/Portfile

```diff
@@ -12,8 +12,8 @@ long_description    shmux is program for executing the same command on \
                     many hosts in parallel. For each target, a child \
                     process is spawned by shmux, and a shell on the target \
                     obtained one of the supported methods: rsh, ssh, or sh. 
-homepage            http://web.taranis.org/shmux/
-
+homepage            https://github.com/shmux/shmux
+license             BSD 3-Clause
```

Comment:
this is not a valid license in MacPorts; please use `BSD` instead

Reply:
Done.  Yeah, 3 clause seems to be "BSD", 4 clause is "bsd-old".